android/app/src/main/AndroidManifest.xml

```diff
@@ -33,16 +35,16 @@
             android:largeHeap="true">
       <meta-data android:name="firebase_analytics_collection_deactivated" android:value="true" />
       <meta-data
-              android:name="com.google.firebase.messaging.default_notification_icon"
-              android:resource="@drawable/ic_stat_status_notification" />
+        android:name="com.google.firebase.messaging.default_notification_icon"
+        android:resource="@drawable/ic_stat_ic_notification" />
         <activity
                 android:name=".MainActivity"
                 android:label="@string/app_name"
                 android:screenOrientation="portrait"
                 android:windowSoftInputMode="adjustResize"
                 android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
                 android:resizeableActivity="false"
-                android:launchMode="singleTask">
+                android:launchMode="singleTop">
```

Comment:
What is this?

Reply:
This was recommended in the react-native-firebase, but I've looked into it and singleTask will functionally work as well. I'll revert this